server/app-engine/gtfsrt.py

```diff
@@ -224,7 +248,8 @@ def get_alert_feed(datastore_client, agency):
     print('- agency: ' + agency)
 
     name = agency + '-alert-feed'
-    alert_feed = cache.get(name)
+    # alert_feed = cache.get(name)
+    alert_feed = None
```

Comment:
What is going on here? Am I missing something or are you disabling caching of service alerts? Please explain.